device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/hsm/HttpsHsmClient.java

```diff
@@ -42,48 +44,15 @@
      */
     public HttpsHsmClient(String baseUrl) throws URISyntaxException
```

Comment:
So with new changes, description of this method should also change?

Reply:
which part of the javadoc header is out of date here?